app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -153,110 +167,19 @@ You can check if you’re eligible for a scholarship and apply by visiting:
 
 [Find teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) in languages and physics.
 
-## Other types of financial support
 
-If you have the right [immigration status](#check-your-immigration-status-and-eligibility-for-financial-support), you could be eligible for the following types of financial help from the UK government.
 
-Tuition fees and maintenance loans are only available for courses where you pay a fee to train to teach.
+## Student finance
```

Comment:
I had to re-read the earlier section as I got a little confused by this sub head as I initially thought bursaries/ scholarships e.t.c. would all be classed as student finance? I don't know if perhaps Student loans might be clearer?

Reply:
Yes good idea, will change